source/administration/config-settings.rst

```diff
@@ -4037,6 +4037,102 @@ Changes to this setting require a server restart before taking effect.
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsSysLogIP": []`` with string array input consisting of an IP address or domain name.            |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Syslog tag
+^^^^^^^^^^^^^^
+The IP address or domain of the syslog server. Use ``localhost`` for local syslog.
+
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsSysLogTag": []`` with string array input consisting of a user-defined field tag.             |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Syslog cert
+^^^^^^^^^^^^^^
+This is the path to the syslog server certificate for TLS connections (``.crt`` or ``.pem``).
+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsSysLogCert": []`` with string array input consisting of a user-defined field tag.             |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: When ``true`` file output is enabled.
+**False**: File output is disabled.
+
++-------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileEnabled": []`` with options ``true`` and ``false``.                               |
++-------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File name
+^^^^^^^^^^^^^^
+This is the path to the output file location.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalAuditSettingsFileName": []`` with string array input consisting of a user-defined name.                     |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+File max size MB
+^^^^^^^^^^^^^^
+This is the maximum size (measured in megabytes) that the file can grow before triggering rotation. The default setting is 100.
```

Comment:
@wiggin77, is there a maximum value here?

Reply:
No maximum other than what an Int can hold.  I don't think a maximum needs to be documented as it would exceed the size of any customer's disk capacity (something like 4 billion megabytes).

Reply:
That makes sense, thank you. :)